bridges/CVEDetailsBridge.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+// CVE Details is a collection of CVEs, taken from the National Vulnerability
+// Database (NVD) and other sources like the Exploit DB and Metasploit. The
+// website categorizes it by vendor and product and attach the CWE category.
+// There is a Atom feed available, but only logged in users can use it,
+// it is not reliable and contain no useful information. This bridge create a
+// sane feed with additional information like tags and a link to the CWE
+// a description of the vulnerability.
+class CVEDetailsBridge extends BridgeAbstract {
+    const MAINTAINER = 'Aaron Fischer';
+    const NAME = 'CVE Details';
+    const CACHE_TIMEOUT = 60*60*6; // 6 hours
+    const DESCRIPTION = 'Report new CVE vulnerabilities for a given vendor (and product)';
+    const URI = 'https://www.cvedetails.com';
+
+    const PARAMETERS = array(array(
+        // The Vendor ID can be taken from the URL
+        'vendor_id' => array(
+            'name' => 'Vendor ID',
+            'type' => 'number',
+            'required' => true,
+        ),
+        // The optional Product ID can be taken from the URL as well
+        'product_id' => array(
+            'name' => 'Product ID',
+            'type' => 'number',
+            'required' => false,
+        ),
```

Comment:
Could you add "exampleValue" for both params please?

Reply:
Sure. I've added PHP as a vendor and product :)